runtime/j9vm/jvm.c

```diff
@@ -3708,7 +3708,7 @@ JVM_LoadLibrary(const char *libName)
 		j9mem_free_memory(libNameConverted);
 	}
 #endif /* defined(WIN32) */
-	Trc_SC_LoadLibrary_Exit(result);
+	Trc_SC_LoadLibrary_Exit2(result);
```

Comment:
Please reuse the old tracepoint name so this file need not change.

Reply:
Reused `Trc_SC_LoadLibrary_Exit`.